### ISSUE TITLE
Issue while installing dependencies

### ISSUE THREAD

**Issue by XmlmXmlmX** [foundation/foundation-emails#297]
When installing Foundation for Emails (`npm install --global foundation-cli`):

```
Installing dependencies...

npm ERR! git clone --template=C:\Users\***\AppData\Roaming\npm-cache\_git-remot
es\_templates --mirror git://github.com/gulpjs/gulp.git C:\Users\***\AppData\Ro
aming\npm-cache\_git-remotes\git-github-com-gulpjs-gulp-git-3014dfdc: Cloning in
to bare repository 'C:\Users\***\AppData\Roaming\npm-cache\_git-remotes\git-git
hub-com-gulpjs-gulp-git-3014dfdc'...
npm ERR! git clone --template=C:\Users\***\AppData\Roaming\npm-cache\_git-remot
es\_templates --mirror git://github.com/gulpjs/gulp.git C:\Users\***\AppData\Ro
aming\npm-cache\_git-remotes\git-github-com-gulpjs-gulp-git-3014dfdc: fatal: rea
d error: Invalid argument
```

What I have tried:
- [x] Cleaning `npm-cache` folder.
- [x] Executing `git clone` commands manually
  - No errors, but when retry `npm install --global foundation-cli` I get a another error:

```
npm ERR! git fetch -a origin (git://github.com/gulpjs/gulp.git) fatal: read error: Invalid argument
```

System: **Windows Server 2012 R2 Standard**
NPM works, `npm install --global gulp-cli` works...

In the `packages.json` file I found this:

```
 "devDependencies": {
    "gulp": "git://github.com/gulpjs/gulp.git#4.0",
```

**Comment by XmlmXmlmX** [comment]
When using `git+https` it works!

https://docs.npmjs.com/files/package.json#git-urls-as-dependencies

**Comment by gakimball** [comment]
So the string is `git+https://github.com/gulpjs/gulp.git#4.0`?

We could update the template's `package.json` to use this.

**Comment by XmlmXmlmX** [comment]
That's right @gakimball 

This is my pull: https://github.com/zurb/foundation-emails-template/pull/14

**Comment by gakimball** [comment]
Merged zurb/foundation-emails-template#14, thanks @XmlmXmlmX!
